app/models/search_builder.rb

```diff
@@ -1,4 +1,10 @@
 # frozen_string_literal: true
 class SearchBuilder < Blacklight::SearchBuilder
   include Blacklight::Solr::SearchBuilderBehavior
+  include Hydra::AccessControlsEnforcement
+
+  def add_access_controls_to_solr_params(*args)
+    return if current_ability.can?(:manage, Valkyrie::Model)
+    apply_gated_discovery(*args)
+  end
```

Comment:
I'm not sure I understand what you mean by it's a `pass-by-default` scheme - can you expand?